docs/get-started/getting-started-alluxio.md

```diff
@@ -0,0 +1,271 @@
+      #       export YOUR_CACHE_DIR=/opt/alluxio/cache
+      #       mkdir -p $YOUR_CACHE_DIR
+      alluxio.worker.tieredstore.level0.dirs.path=/YOUR_CACHE_DIR
+
+      # The quotas for all storage directories in top storage tier 0
+      # For example, set the quota to 100G.
+      alluxio.worker.tieredstore.level0.dirs.quota=100G
+
+      # The path to the domain socket. Short-circuit reads make use of a UNIX domain
+      # socket when this is set (non-empty). This is a special path in the file system
+      # that allows the client and the AlluxioWorker to communicate. You will need to
+      # set a path to this socket. The AlluxioWorker needs to be able to create the
+      # path. If alluxio.worker.data.server.domain.socket.as.uuid is set, the path
+      # should be the home directory for the domain socket. The full path for the domain
+      # socket with be {path}/{uuid}.
+      # For example,
+      #      export YOUR_DOMAIN_SOCKET_PATH=/opt/alluxio/domain_socket
+      #      mkdir -p YOUR_DOMAIN_SOCKET_PATH
+      alluxio.worker.data.server.domain.socket.address=/YOUR_DOMAIN_SOCKET_PATH
+      alluxio.worker.data.server.domain.socket.as.uuid=true
+
+      # Configure async cache manager
+      # When large amounts of data are expected to be asynchronously cached concurrently,
+      # it may be helpful to increase below async cache configuration to handle a higher
+      # workload.
+
+      # The number of asynchronous threads used to finish reading partial blocks.
+      alluxio.worker.network.async.cache.manager.threads.max=64
+
+      # The maximum number of outstanding async caching requests to cache blocks in each
+      # data server.
+      alluxio.worker.network.async.cache.manager.queue.max=2000
+      ############################################################################
+
+      ########################### Client properties ##############################
+      # When short circuit and domain socket both enabled, prefer to use short circuit.
+      alluxio.user.short.circuit.preferred=true
+      ############################################################################
+
+      # Running Alluxio locally with S3
+      # Optionally, to reduce data latency or visit resources which are separated in
+      # different AWS regions, specify a regional endpoint to make aws requests.
+      # An endpoint is a URL that is the entry point for a web service.
+      #
+      # For example, s3.cn-north-1.amazonaws.com.cn is an entry point for the Amazon S3
+      # service in beijing region.
+      alluxio.underfs.s3.endpoint=<endpoint_url>
+
+      # Optionally, specify to make all S3 requests path style
+      alluxio.underfs.s3.disable.dns.buckets=true
+      ```
+
+      For more explanations of each configuration, please refer to
+      [Alluxio Configuration](https://docs.alluxio.io/os/user/stable/en/reference/Properties-List.html)
+      and [Amazon AWS S3](https://docs.alluxio.io/os/user/stable/en/ufs/S3.html).
```

Comment:
I think we should also add a statement about security. I assume with this default setup there is no security enabled so we should add a message stating that and point to the alluxio security page.

Reply:
done

Reply:
I'm missing it, what was added for this?  I don't see the word security in this doc.

Reply:
I would us to say something about the instructions start a insecure Alluxio cluster.  User is responsible for adding security as needed, see Alluxio security doc here.....

Reply:
Sorry, my misunderstanding, Done